apps/taskmasters/flowlord/taskmaster.go

```diff
@@ -42,13 +44,16 @@ type taskMaster struct {
 	*workflow.Cache
 	cron  *cron.Cron
 	slack *slack.Slack
+
+	refreshChan chan struct{}
 }
 
 type stats struct {
-	RunTime    string            `json:"runtime"`
-	NextUpdate string            `json:"next_cache_update"`
-	Workflow   map[string]int    `json:"workflow"`
-	Entries    map[string]cEntry `json:"cron"`
+	RunTime    string `json:"runtime"`
+	NextUpdate string `json:"next_update"`
+	LastUpdate string `json:"last_update"`
+
+	Workflow map[string]map[string]cEntry `json:"workflow"`
```

Comment:
a `map[string]map[string]cEntry` this is a little confusing.  Maybe  a comment here to define what this actually means? and what each key represents?